### ISSUE TITLE
SacRT Go Live Checklist

### ISSUE THREAD

**Issue by thekaveman** [cal-itp/benefits#1194]
See latest on https://github.com/cal-itp/benefits/issues/1240

* [x] Obtain list of discount product(s) from SacRT
* [x] QA: [Register discount products](https://github.com/cal-itp/benefits-secrets/blob/main/littlepay/bin/customer/assign-product-group.sh) to test group `d9b41fcd-eb89-41b1-a492-e7dc1598334f`
* [x] Prod: [Register discount products](https://github.com/cal-itp/benefits-secrets/blob/main/littlepay/bin/customer/assign-product-group.sh) to test group, "(Prod) Test Group 0": `80271c54-29f6-465c-ad33-0f6ef18a15be`
   * [x] Create assignments listing for documentation
* [x] Update `dev` and `test` data migrations for SacRT 
* [x] Test standard discount on the ground in SacRT
* [ ] Test cap discount on the ground in SacRT
* [ ] [Create new LP group](https://github.com/cal-itp/benefits-secrets/blob/main/littlepay/bin/customer/post-group.sh) in SacRT `prod`
* [ ] [Register discount products](https://github.com/cal-itp/benefits-secrets/blob/main/littlepay/bin/customer/assign-product-group.sh) to new group
  * [ ] Update assignments listing for documentation
* [ ] Update `prod` config with SacRT agency, eligibility type, new group, help link, phone number, etc.
* [ ] [Delete test group from products](https://github.com/cal-itp/benefits-secrets/blob/main/littlepay/bin/customer/delete-product-group.sh)
* [ ] [Delete test group](https://github.com/cal-itp/benefits-secrets/blob/main/littlepay/bin/customer/delete-group.sh)

Resources

- [Old SacRT prep doc](https://docs.google.com/document/d/1D6li-i8ScY3D1-P-0qyh_9-UHwD6JGRyvf77V7LZncM/edit)
- [Checklist for go-live with MST+Login.gov](https://github.com/cal-itp/benefits-secrets/issues/13)

**Comment by thekaveman** [comment]
Blocked waiting for SacRT product information

**Comment by thekaveman** [comment]
Copying the MST Go-Live Checklist as a reference from https://github.com/cal-itp/benefits-secrets/issues/13

- [x] cal-itp/benefits#615
- [x] cal-itp/benefits#621
- [x] Verify redirects for Login.gov flow
- [x] Cal-ITP verifies logs (Azure), events (Amplitude)
- [x] Create a new product group for testing purposes
- [x] Configure `prod` data migrations for MST with the test product group ⬆️ 
- [x] Swap back to Login.gov sandbox
- [x] Create Login.gov sandbox account for MST
- [x] MST live test in the field
- [x] MST verifies taps, discounts (Merchant Portal)
- [x] Cal-ITP verifies logs (Azure), events (Amplitude)
- [x] Swap back to full `prod` configuration

**Comment by o-ram** [comment]
Product code: DISC001

Information received from SacRT confirms that product is enabled/active.

**Comment by machikoyasuda** [comment]
SacRT's contactless help page link: sacrt.com/contactless

**Comment by thekaveman** [comment]
Waiting on SacRT to fix a product configuration for their capping products. [Slack thread with details](https://cal-itp.slack.com/archives/C013W8RUAMU/p1680736086236229).

**Comment by o-ram** [comment]
SacRT reported that the fix has been made to the product configuration for the capping products! I believe this means we are ready to do another round of on-the-ground testing. Before kicking that off, is there anything the dev team would like/needs to check from our side? SacRT just filled out the missing required field and touched nothing else, so there shouldn't be any new products but we may want to double check that to be safe.

**Comment by angela-tran** [comment]
Hi @o-ram , I checked on SacRT's products today. 

I see the two capping products that we used in the last round of production validation (the ones named `Senior Discount 90 min Cap` and `Senior Discount Daily Cap`). However, I also see two new product IDs, one with the name `Senior Discount 90 min Cap` and the other with `Senior Discount Daily Cap`.

I would guess that these newer products are the ones with the required field filled out? They have not been associated to the group yet.

**Comment by angela-tran** [comment]
Sounds good @o-ram. I'm going to associate the newer products with the group and **also** un-associate the older capping products from the group. This is so we can be sure the newer products are what worked. I'll let you know here when I'm done!

**Comment by angela-tran** [comment]
Done making those changes 👍

**Comment by o-ram** [comment]
Thank you @angela-tran! Are we now ready for Jarrett and John to go out and test again?

**Comment by angela-tran** [comment]
One more check I think we could do is to ask for screenshots from SacRT of the capping products, just to double-check.

**Comment by o-ram** [comment]
Update: Jarrett and John are testing on Fri. 06/02.

**Comment by o-ram** [comment]
See latest on #1240

**Comment by thekaveman** [comment]
See latest and why this ticket remains `Paused` and not assigned to any Sprint: https://github.com/cal-itp/benefits/issues/1240#issuecomment-1885466351

**Comment by indexing** [comment]
@o-ram In a meeting with the payments team last week we learned that SacRT is removing all hardware validators on light rail that they implemented during the pilot. Thus, we will soon no longer be able to test the pilot implementation. 

Going forward, any production validation will need to be done using hardware implemented as part of the new contracts. 

Additionally, we are unsure if the current Littlepay configuration that was implemented as part of the pilot on light rail will be utilized with the new hardware. It may be that the new contracts require an entirely new instance for SacRT on the Littlepay platform. @jenro1 can you confirm?

Given that the pilot is effectively over, I propose we close this issue. If you agree @o-ram, please close.

**Comment by jenro1** [comment]
Hi @indexing - I'm going to move this chat to Benefits in Slack if ok, to include some of the Payments team re: the LP question. Thanks!

**Comment by thekaveman** [comment]
As the Pilot in SacRT is over, we'll be tracking a new effort to officially go live with SacRT's contactless implementation.
